packages/table/src/table.scss

```diff
@@ -146,11 +146,3 @@ $menu-z-index: $row-z-index + 1 !default;
   display: inline-block;
   position: absolute;
 }
-
-.#{$ns}-table-scrollbar-measure {
```

Comment:
this class was apparently unused. I guess we removed whatever code we had to detect scrollbar width.